rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -83,16 +83,6 @@ Player::Player(const std::string & node_name, const rclcpp::NodeOptions & node_o
   throw rclcpp::exceptions::UnimplementedError();
 }
 
-Player::Player(
-  const rosbag2_storage::StorageOptions & storage_options,
-  const rosbag2_transport::PlayOptions & play_options,
-  const std::string & node_name,
-  const rclcpp::NodeOptions & node_options)
-: Player(std::make_unique<rosbag2_cpp::Reader>(),
-    storage_options, play_options,
-    node_name, node_options)
-{}
-
```

Comment:
Why this change? This is currently being used within the rosbag2_bag_v2 plugins

Reply:
Oh - this was a bad rebase - good catch